services/minespace-web/src/routes/Routes.tsx

```diff
@@ -6,23 +6,28 @@ import React, { Suspense } from "react";
 import AuthenticationGuard from "@/HOC/AuthenticationGuard";
 import * as routes from "@/constants/routes";
 import Loading from "@/components/common/Loading";
+import ColumnWrapper from "@/HOC/ColumnWrapper";
 
 const Routes = () => (
   <Suspense fallback={<Loading></Loading>}>
     <Switch>
       {/* PUBLIC ROUTES */}
-      <Route exact path={routes.HOME.route} component={routes.HOME.component} />
-      <Route exact path={routes.RETURN_PAGE.route} component={routes.RETURN_PAGE.component} />
+      <Route exact path={routes.HOME.route} component={ColumnWrapper()(routes.HOME.component)} />
+      <Route
+        exact
+        path={routes.RETURN_PAGE.route}
+        component={ColumnWrapper()(routes.RETURN_PAGE.component)}
+      />
       {/* PRIVATE ROUTES */}
       <Route
         exact
         path={routes.MINES.route}
-        component={AuthenticationGuard()(routes.MINES.component)}
+        component={ColumnWrapper()(AuthenticationGuard()(routes.MINES.component))}
```

Comment:
The column wrapper solution wasn't quite _ideal_ but at least I only had to make changes in this file (+ create the component) instead of every route file in this file.